app.js

```diff
@@ -1,34 +1,34 @@
 // ℹ️ Gets access to environment variables/settings
 // https://www.npmjs.com/package/dotenv
-require('dotenv/config');
+require("dotenv/config");
 
 // ℹ️ Connects to the database
-require('./db');
+require("./db");
 
 // Handles http requests (express is node js framework)
 // https://www.npmjs.com/package/express
-const express = require('express');
+const express = require("express");
 
 // Handles the handlebars
 // https://www.npmjs.com/package/hbs
-const hbs = require('hbs');
+const hbs = require("hbs");
 
 const app = express();
 
 // ℹ️ This function is getting exported from the config folder. It runs most middlewares
-require('./config')(app);
+require("./config")(app);
 
 // default value for title local
-const projectName = 'lab-express-cinema';
-const capitalized = string => string[0].toUpperCase() + string.slice(1).toLowerCase();
+const projectName = "lab-express-cinema";
+const capitalized = (string) =>
+  string[0].toUpperCase() + string.slice(1).toLowerCase();
 
 app.locals.title = `${capitalized(projectName)}- Generated with Ironlauncher`;
 
 // 👇 Start handling routes here
-const index = require('./routes/index');
-app.use('/', index);
-
+const index = require("./routes/index");
+app.use("/", index);
```

Comment:
Esto puede ayudarte solo que todas tus rutas en ./routes/movies van a necesitar el prefijo /movies cuando accedes en el navegador pero no en el archivo, esas permanecen este es un prefijo solamente
```suggestion
app.use("/movies", require("./routes/movies"));
```